examples/HighchartsGanttWithHighstockNavigation/App.js

```diff
@@ -0,0 +1,88 @@
+import React, {Component} from 'react';
+import Highcharts from 'highcharts/highcharts-gantt';
+import {
+  GanttSeries,
+  HighchartsGanttChart,
+  Subtitle,
+  Title,
+  withHighcharts,
+  YAxis
+} from 'react-jsx-highcharts-gantt';
+import ExampleCode from '../utils/ExampleCode';
+import code from './exampleCode';
+import {Scrollbar, RangeSelector, Navigator} from 'react-jsx-highstock';
+
+
+class App extends Component {
+
+  render() {
+
+
+    return (
+      <div className="app">
+        <HighchartsGanttChart>
+          <Title>Gantt Chart with Highstock Navigation</Title>
+          <YAxis
+            uniqueNames={true}
+          />
+          <Navigator enabled={true}>
```

Comment:
As stated in the readme, [`enabled=true`](https://github.com/whawker/react-jsx-highcharts#example) is assumed on mount. So this is redundant

Reply:
OK, I've removed these.